packages/blade/src/components/Form/Selector/SelectorInput.web.tsx

```diff
@@ -50,12 +52,12 @@ const StyledInput = styled.input<HoverProps>(({ theme, isChecked, isDisabled, is
   },
 }));
 
-const CheckboxInput = ({
+const SelectorInput = ({
   inputProps,
   isChecked,
   isDisabled,
   isNegative,
-}: HoverProps & { inputProps: InputProps }): React.ReactElement => {
+}: HoverProps & { inputProps: any }): React.ReactElement => {
```

Comment:
Oh no :(

Reply:
No other way around?

Reply:
Could be, but fine for now. Anyways this component is only shared in Checkbox & radio for now. which has similar APIs